source/geometry_model/ellipsoidal_chunk.cc

```diff
@@ -600,6 +600,16 @@ namespace aspect
 
     template <int dim>
     double
+    EllipsoidalChunk<dim>::height_above_original_surface(const Point<dim> &position) const
+    {
+      AssertThrow(false, ExcMessage("The topography postprocessor does not recognize the geometry model. "
```

Comment:
You can also support this by using the topo plugin and to get the topo height and adding it to the manifold.pull_back(position)[2] (this is the height above the reference surface - topo).

Reply:
Sorry I didn't quite get what you mean. I'm not really familiar with the ellipsoidal chunk model, should the function return just -manifold.pull_back(position)[2] ?

Reply:
I think you should be able to get it working with the following code (although I haven't tested it):
```
const Point<dim> phi_theta_h = manifold.pull_back(position);
Point<dim-1> phi_theta;
const double rad_to_degree = 180/numbers::PI;
if (dim == 3)
  phi_theta = Point<dim-1>(phi_theta_h[0] * rad_to_degree,phi_theta_h[1] * rad_to_degree);
return phi_theta_h[2] + topography->value(phi_theta).
```

If it doesn't work, you can leave it as it is if you want. It can be added in an other pull request.